app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -286,7 +286,7 @@ def tasks_list(filtered_scope)
         filtered_scope = search_scope.select('DISTINCT foreman_tasks_tasks.*')
 
         ordering_params = {
-          sort_by: params[:sort_by] || 'started_at',
+          sort_by: 'foreman_tasks_tasks.' + (params[:sort_by] || 'started_at'),
```

Comment:
Couldn't we instead up our game, and use `scope.order(sort_by => sort_order)` in the `ordering_scope`?
I believe than rails will do the table prefixing by itself.

Reply:
Maybe it needs to be `(sort_by.to_sym => sort_order.downcase.to_sym)`

Reply:
Good point